src/ceph-volume-zfs/setup.cfg

```diff
@@ -0,0 +1,22 @@
+[bumpversion]
```

Comment:
does it make sense to keep versions if this is going on the ceph source tree?

Reply:
@alfredodeza 
This also came with the framework, I can delete that, I'd expect.